aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -79,13 +79,39 @@ public static IWebHost BuildWebHost(string[] args) =>
         ...
 ```
 
-The ASP.NET Core Module generates a dynamic port to assign to the back-end process. `CreateDefaultBuilder` calls the <xref:Microsoft.AspNetCore.Hosting.WebHostBuilderIISExtensions.UseIISIntegration*> method, which picks up the dynamic port and configures Kestrel to listen on `http://localhost:{dynamicPort}/`. This overrides other URL configurations, such as calls to `UseUrls` or [Kestrel's Listen API](xref:fundamentals/servers/kestrel#endpoint-configuration). Therefore, calls to `UseUrls` or Kestrel's `Listen` API aren't required when using the module. If `UseUrls` or `Listen` is called, Kestrel only listens on the ports specified when running the app without IIS.
+The ASP.NET Core Module generates a dynamic port to assign to the backend process. `CreateDefaultBuilder` calls the <xref:Microsoft.AspNetCore.Hosting.WebHostBuilderIISExtensions.UseIISIntegration*> method, which picks up the dynamic port and configures Kestrel to listen on `http://127.0.0.1:{dynamicPort}/`. This configuration replaces other URL configurations provided by:
```

Comment:
I guess there's no way to localize dynamicPort. Should it be dynamic Port?

cc @JasonCard @tdykstra 	@scottaddie 		- it would be nice is we could mark "dynamic port" to be localized.

Reply:
I thought no code (designated in markdown) was localized.

We could start treating links like we treat files and do it this way ...

> \*http:\//127.0.0.1:{Dynamic Port}/\*

*http:\//127.0.0.1:{Dynamic Port}/*

... or no style ...

> http:\//127.0.0.1:{Dynamic Port}/

http:\//127.0.0.1:{Dynamic Port}

... or will tags work? (allow localization and result in code style) ...

> \<code>http:\//127.0.0.1:{Dynamic Port}/\</code>

Reply:
... which picks up the dynamic port and configures Kestrel to listen on it at the localhost IP address. For example, if the dynamic port is 1234, the IP address that Kestrel listens on is `http://127.0.0.1:1234/`.

Reply:
I'll try something like that on the next commit, but I'm trying to avoid use the subordinate conjunction like that. Let's see how it looks on the next pass.